examples/using-remark/src/utils/typography.js

```diff
@@ -131,15 +131,18 @@ const options = {
         overflow: `auto`,
       },
       ".gatsby-highlight pre[class*='language-']": {
-        backgroundColor: `transparent`,
-        borderRadius: 0,
         margin: 0,
-        padding: 0,
         overflow: `initial`,
         float: `left`,
         minWidth: `100%`,
         textShadow: `none`,
       },
+      // Override `inline code block` background color from prismjs
+      "code[class*='language-']": {
+        backgroundColor: `transparent !important`,
+        borderRadius: `0 !important`,
+        padding: `0 !important`
+      },
```

Comment:
The `!important` part is code smelly. You also removed those rules from `.gatsby-highlight pre[class*='language-']` selector and used just `code` vs `pre`, which is also a bit confusing.

But I will checkout and see if there might be some nicer solution than using `!important`